### PR TITLE
Quick cleanup

### PR DESCRIPTION
**Pull request by ydidwania**
- [x] use `const` instead of  `let ` and `var` in imports
- [x] add a user-config-example.yml
- [ ] add .taskcluster.yml 
- [x] remove no longer relevant comments / add new comments
- [x] add a ping api

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -51,6 +51,17 @@ var parseAndValidateBindings = function(bindings) {
   });
 };
 
+builder.declare({
```
Comment (imbstack) [inline src/api.js]: `taskcluster-lib-api` comes with ping built in!

File: user-config-example.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+test:
+   pulse:
+    username:       !env PULSE_USERNAME
```
Comment (imbstack) [inline user-config-example.yml]: We generally set the examples to be `...`. See https://github.com/taskcluster/taskcluster-queue/blob/master/user-config-example.yml for example!

File: Readme.md [hunk 3]
```diff
@@ -48,7 +48,8 @@ This is done by setting the `id` of each event to `-` and returning `204 : No in
 
 Install npm dependencies using `yarn` and run `yarn test` to run the tests. 
 To build it locally you need to use `NODE_ENV='test' TASKCLUSTER_ROOT_URL='localhost:12345' node src/main.js server` 
-This will use the pulse credentials in `user-config.yml`. Set `DEBUG: events:*` for additional debugging information.
+This will use the pulse credentials in the test profile of `user-config.yml` (see user-config-example.yml). Set `PULSE_VHOST: /`  to use internally.
```
Comment (imbstack) [inline Readme.md]: In these cases, `DEBUG` is an env var so it should be set with `=` and `PULSE_VHOST` should be set in user-config so it is `pulse.vhost: '/'`. What do we mean by internally btw?

**Review by imbstack — APPROVED** [review]
Looks good! Just a couple of nits.

File: user-config-example.yml (hunk 2 above)
Comment (ydidwania) [reply user-config-example.yml]: ah, i will correct it. And maybe also update the `user-config-example.yml` in tc-queue

File: Readme.md (hunk 3 above)
Comment (ydidwania) [reply Readme.md]: Yeah it should be `=` .Sorry i didnt catch that earlier.
I mean when using with `pulse.mozilla.org`

File: user-config-example.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+test:
+   pulse:
+    username:       '...'
+    password:       '...'
+    hostname:       '...'
+    vhost:          '...'
```
Comment (helfi92) [inline user-config-example.yml]: Add newline.

File: Readme.md [hunk 5]
```diff
@@ -48,7 +48,8 @@ This is done by setting the `id` of each event to `-` and returning `204 : No in
 
 Install npm dependencies using `yarn` and run `yarn test` to run the tests. 
 To build it locally you need to use `NODE_ENV='test' TASKCLUSTER_ROOT_URL='localhost:12345' node src/main.js server` 
-This will use the pulse credentials in `user-config.yml`. Set `DEBUG: events:*` for additional debugging information.
+This will use the pulse credentials in the test profile of `user-config.yml` (see user-config-example.yml). Set `pulse.vhost:/` in user-config.yml when using with`pulse.hostname:pulse.mozilla.org`.
```
Comment (helfi92) [inline Readme.md]: Add a space after `pulse.vhost:` and `pulse.hostname:`.

**Review by helfi92 — APPROVED** [review]

File: src/api.js [hunk 6]
```diff
@@ -60,7 +60,6 @@ builder.declare({
   name: 'connect',
   description: 'Connect to receive messages',
   stability: APIBuilder.stability.experimental,
-  // Add input validation yml
```
Comment (imbstack) [inline src/api.js:63]: Why not do input validation?

Comment (ydidwania) [reply src/api.js:63]: Input validation is already done here - https://github.com/taskcluster/taskcluster-events/blob/master/src/api.js#L28

**Comment by helfi92** [comment]
I believe a rebase is needed.

**Comment by djmitche** [comment]
@ydidwania did you want to fix this up?

**Comment by ydidwania** [comment]
@djmitche Yes, i did want to, but couldn't find the time. Will it be okay if I do it by the end of this month? 
This semester gets over on 28th

**Comment by djmitche** [comment]
Sure, that sounds good!

**Comment by ydidwania** [comment]
@djmitche I did the rebase and I think I resolved most of the comments on this PR. Can you take another look? 
Let's do the tc-github integration in another PR?

**Review by djmitche — APPROVED** [review]
Looks good!

**Comment by djmitche** [comment]
deployed and looks to be running smoothly.. thanks!

**Pull request merged**